### PR TITLE
[bitnami/milvus] feat: config external S3 tls client certs settings (…

### PR DESCRIPTION
**Pull request by chenraoCR**
### Description of the change

config externalS3 tls settings

### Benefits

allow users to config minio tls client certs when connect external minio running with tls

### Applicable issues

- feat: https://github.com/bitnami/charts/issues/26111

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by chenraoCR** [comment]
this feature is just like https://github.com/bitnami/charts/issues/26110, please check and approve

File: bitnami/milvus/templates/_helpers.tpl [hunk 1]
```diff
@@ -816,14 +814,24 @@ Init container definition for waiting for the database to be ready
 
       check_s3() {
           local -r s3_host="${1:-?missing s3}"
-          if curl --max-time 5 "${s3_host}" | grep "RequestId"; then
+          local params_cert=""
+
+          if echo $s3_host | grep https; then
```
Comment (jotamartos) [inline bitnami/milvus/templates/_helpers.tpl:819]: What about this?

```suggestion
          {{- if and .Values.externalS3.tls.enabled }}
```

**Review by jotamartos — COMMENTED** [review]
Thanks for your contribution. Can you take a look at my new comment? I didn't notice that in the first review. Thanks

File: bitnami/milvus/Chart.yaml [hunk 2]
```diff
@@ -48,4 +48,4 @@ maintainers:
 name: milvus
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/milvus
-version: 8.2.0
+version: 8.2.1
```
[the remark this answers is not in the document]
Comment (jotamartos) [reply bitnami/milvus/Chart.yaml]: Version is now updated

File: bitnami/milvus/templates/_helpers.tpl (hunk 1 above)
Comment (chenraoCR) [reply bitnami/milvus/templates/_helpers.tpl]: sure, already updated

**Review by jotamartos — APPROVED** [review]

**Pull request merged**